contract/Makefile

```diff
@@ -10,12 +10,20 @@ ATOM=000000$(ATOM_DENOM)
 
 # your deployer address made from make add-address
 # ADDR=agoric1k78s7qz7rxy8afyjrqk3dntg8m83zaw3upe60p
-ADDR=agoric125d8s0tamx50gpqdd3m7f5gtkzkcnjs0ehsujr
+USERNAME ?= user1
 
-PROVISION_POOL_ADDR=agoric1megzytg65cyrgzs6fvzxgrcqvwwl7ugpt62346
+ADDR := $(shell kubectl exec -i agoriclocal-genesis-0 -c validator -- sh -c '\
+    if ! agd keys show $(USERNAME) >/dev/null 2>&1; then \
```

Comment:
Is there some reason we don't use a fixed mnemonic and address for user1?

Reply:
Plan to work on this in a subsequent PR unless someone else gets to it before me.